Sources/Offline/OTT/OfflineManager+OTT.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  OfflineManager+OTT.swift
+//  KalturaPlayer
+//
+//  Created by Nilit Danan on 7/19/20.
+//
+
+import Foundation
+import PlayKit
+import PlayKitProviders
+
+extension OfflineManager {
+    
+    private func retrieveMediaEntry(mediaOptions: OTTMediaOptions, callback: @escaping (Error?, PKMediaEntry?) -> Void) {
+        let phoenixMediaProvider = mediaOptions.mediaProvider()
+        
+        let sessionProvider = SimpleSessionProvider(serverURL: KalturaOTTPlayerManager.shared.serverURL,
+                                                    partnerId: KalturaOTTPlayerManager.shared.partnerId,
+                                                    ks: mediaOptions.ks)
+        
+        // Add a function to the Utils
+        var referrer: String = "app://"
+        if let appId = Bundle.main.bundleIdentifier {
```

Comment:
The app should be able to set the referrer, then it overrides this default.

Reply:
Changed, It will be set in the setup func as an optional.
And now OfflineManager can fetch it from there.